Standards/scs-0218-v1-container-registry-for-scs-standard-implementation.md

```diff
@@ -11,65 +11,35 @@ A container registry is an infrastructure service to enable storing and accessin
-discussed, but overall CSP could offer a private container registry as a service or
-CSP could offer a recipe (maintained by SCS) for customers to deploy the private
-registry themselves utilizing CSP infrastructure. In both cases, the private
-container registry should fulfill a set of needed requirements e.g. for security and
-privacy.
-
-The idea and purpose of this document is to specify what requirements a
-specific technical container registry implementation (i.e. software solution) needs to
-fulfill in the context of SCS.
-
-Another purpose is the selection of an appropriate container registry
-implementation that meets all defined requirements to make architectural
-decision on what implementation will be used by the SCS.
-
-## Design considerations
+## Terminology
 
-There are numerous features to look for when evaluating a container registry solution.
-Our decision process goes through two main stages:
+Cloud Service Provider (abbr. CSP)
+  Entity that provides scalable computing resources
 
-1. [OSS health check](#oss-health-check)
-2. [Required and desirable features check](#required-and-desirable-features-check)
+Cloud Native Computing Foundation (abbr. CNCF)
+  Organization that hosts and develops open source projects for cloud native computing
 
-The open-source software (OSS) health check is the first filter stage. This stage is
-crucial in the context of SCS and container registry implementation should pass it to
-promote itself to the second consideration stage. The second stage provides an overview
-over the feature set of open source container registry implementations and map out
-requirements (and nice-to-haves) against it to have a well-documented decision.
-
-Note: Keep in mind that at the time of writing this document, we've made our best effort
-to survey the container registry landscape based on publicly available materials.
-If you find something outdated (with respect to the time of writing this document) or
-outright erroneous, please submit a PR or raise an issue, and we'll fix it right away.
+## Motivation
 
-### OSS health check
+In order to provide a usable, complete experience for the SCS reference implementation, it must be decided
+on a registry in accordance with requirements set by the ["Requirements for container registries"] standard
+as well as other dependencies set by the SCS project, including the [OSS requirements](https://github.com/SovereignCloudStack/standards/blob/main/Drafts/OSS-Health.md).
 
-This section evaluates the health of the open-source projects that were selected from
-the currently available solutions. The container registry software must fulfill all OSS
-health checks defined by the [OSS-Health](https://github.com/SovereignCloudStack/standards/blob/main/Drafts/OSS-Health.md)
-document. The main health checks are:
+This document should finally lead to a decision about the container registry used in the implementation.
```

Comment:
consider: This document should finally lead to a decision about the container registry used as a reference implementation of the SCS container registry.

Reply:
Changed in https://github.com/SovereignCloudStack/standards/pull/458/commits/e73ed463c4d03e128dcfa7f1437bfa7131de3f93